driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -722,12 +759,30 @@ public boolean onTimeout(Connection connection, long latency, int retryCount) {
             }
 
             Host queriedHost = current;
-            OperationTimedOutException timeoutException = new OperationTimedOutException(connection.address);
+
+            // This cannot be OperationTimedOutException because
```

Comment:
Not sure I understand from this comment the reason behind the change, can you provide some more info around this change?  Just for my own benefit :)